pkg/skaffold/schema/latest/config.go

```diff
@@ -510,9 +510,13 @@ type HelmRelease struct {
 	UseHelmSecrets bool `yaml:"useHelmSecrets,omitempty"`
 
 	// Remote specifies whether the chart path is remote, or exists on the host filesystem.
-	// `remote: true` implies `skipBuildDependencies: true`.
+	// `remote: true` implies `skipBuildDependencies: true` and `upgradeOnChange: false`.
+	// `remote: false` implies `upgradeOnChange: true`.
 	Remote bool `yaml:"remote,omitempty"`
```

Comment:
These comments seem a bit complex.  Would it make sense to move the explanation of the defaults to `skipBuildDependencies` and `upgradeOnChange`?

Reply:
Makes sense.